Firestore/Swift/Tests/Integration/PipelineApiTests.swift

```diff
@@ -0,0 +1,399 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import XCTest
+
+import FirebaseFirestore
+
+final class PipelineTests: FSTIntegrationTestCase {
```

Comment:
I do not think we need to keep this file around. It serves a purpose as a way to check for compilation errors, but we can just move them to the test below and actually run them.

Reply:
I agree these tests here can be convert to integration tests which verify the result. However, I would like the Swift integration test file consistent with other platforms in order for easy maintenance in the future.